app/controllers/coach/lessons_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Coach
+  class LessonsController < ApplicationController
+    before_action :set_lesson, only: [:show, :edit, :update, :destroy]
+
+    def index
+      @lessons = Lesson.where(current_user.id = coach_id)
+    end
+
+    def new
+      # @user = current_user
+      @lesson = Lesson.new
+    end
+
+    def create
+      @lesson = Lesson.new(lesson_params)
+      # @user = current_user
+      @lesson.coach_id = current_user
```

Comment:
is this line required? Current;y we don't really need to verify the coach.

Reply:
Hm i wanted to add current user as the coach_id since it is a required attribute before saving

Reply:
Oh right! sorry about it!